client/app/scripts/liveblog-edit/posts.service.ts

```diff
@@ -307,17 +307,31 @@ const postsService = (api, $q, _userList, session) => {
 
             // save every items
             _.each(items, (itemParam) => {
-                const item = {
-                    blog: blogId,
-                    text: itemParam.text,
-                    meta: itemParam.meta,
-                    group_type: itemParam.group_type,
-                    item_type: itemParam.item_type,
-                    commenter: itemParam.meta && itemParam.meta.commenter,
-                    syndicated_creator: itemParam.syndicated_creator,
-                };
-
-                dfds.push(api.items.save(item));
+                switch (itemParam.item_type) {
+                case 'poll': {
+                    const poll = {
+                        blog: blogId,
+                        poll_body: itemParam.poll_body,
+                    };
+
+                    dfds.push(api.polls.save(poll));
```

Comment:
I know this variable `dfds` was already there but, could you please rename to something that has some meaning?